tests/sample.yaml

```diff
@@ -0,0 +1,52 @@
+test_duration: 10 # [minutes] + 10 minutes (more than 1800 keep-alive)
+prepare_write_cmd: "cassandra-stress write no-warmup cl=QUORUM n=3000 -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop seq=1..3000"
+stress_read_cmd: ["cassandra-stress read no-warmup cl=QUORUM duration=10m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1 -pop 'dist=gauss(1..3000,1500,150)'"]
+
+stress_cmd: ["cassandra-stress mixed cl=QUORUM duration=10m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=2 -pop seq=1..3000 -log interval=5"
+             ]
+n_db_nodes: 3
+n_loaders: 1
+#round_robin: 'true' # when num loader more than 1
+n_monitor_nodes: 1
+keyspace_num: 1
+user_prefix: 'sample-configuration-tst'
+nemesis_class_name: 'NoOpMonkey' # no nemesis will be applied
+nemesis_interval: 5
+nemesis_during_prepare: 'true'
+failure_post_behavior: destroy
+space_node_threshold: 167000
+
+# scylla-manager configuration
+# if running on aws and use_mgmt is true, the monitor image should not contain scylla
+use_mgmt: true
+mgmt_port: 10090
+scylla_repo_m: 'http://repositories.scylladb.com/scylla/repo/f4a2920f80c4bf178217c2553ad65ad7/centos/scylladb-2017.1.repo'
+scylla_mgmt_repo: 'http://downloads.scylladb.com.s3.amazonaws.com/manager/rpm/unstable/centos/branch-1.0/latest/scylla-manager.repo'
+
+
+# email definitions to send regression report
+send_email: true
+email_recipients: ['bentsi@scylladb.com']
+
+backends: !mux
+    aws: !mux
+        instance_provision: 'spot_low_price' # Allowed values: on_demand|spot_fleet|spot_low_price|spot_duration
+        cluster_backend: 'aws'
+        user_credentials_path: '~/.ssh/scylla-qa-ec2'
```

Comment:
`scylla-qa-ec2` is used for builder machines. Can we use another shared different key for test instances?
Or we allow all internal SCT users to login the builders?